deepchem/feat/coulomb_matrices.py

```diff
@@ -3,21 +3,27 @@
 
 See Montavon et al., _New Journal of Physics_ __15__ (2013) 095003.
 """
-__author__ = "Steven Kearnes"
-__copyright__ = "Copyright 2014, Stanford University"
-__license__ = "MIT"
-
 import numpy as np
 import deepchem as dc
-from deepchem.feat import Featurizer
+from deepchem.feat.base_classes import MolecularFeaturizer
 from deepchem.utils import pad_array
 from deepchem.feat.atomic_coordinates import AtomicCoordinates
 
 
-class BPSymmetryFunctionInput(Featurizer):
-  """
-  Calculate Symmetry Function for each atom in the molecules
-  Methods described in https://journals.aps.org/prl/pdf/10.1103/PhysRevLett.98.146401
+class BPSymmetryFunctionInput(MolecularFeaturizer):
+  """Calculate Symmetry Function for each atom in the molecules
+
+  This method is described in [1]_ 
```

Comment:
it is better to add parameters doc string.

Reply:
Good sugggestion! I've added these in.